src/core/internal/fetch/fetchClient.ts

```diff
@@ -35,15 +40,15 @@ export class FetchClient implements HttpClient {
         config?: { headers: Record<string, string> },
         data?: unknown,
     ): Promise<T> {
-        let body;
+        let body = '{}';
```

Comment:
Why do we need to define value here? `body` should be undefined for `GET` or `HEAD`. Also we do not provide `body` for `DELETE` methods as well.

Reply:
964bc8c

Reply:
scratch that, ref d726edc instead